tests/test_rdf_writer.py

```diff
@@ -38,8 +39,30 @@ def test_accept_provided_doc_node(temporary_file_path) -> None:
     assert parsed_document.spdx_id is None
 
 
+def test_external_package_references(temporary_file_path) -> None:
+    document: Document = minimal_document()
+    package: Package = document.packages[0]
+    first_ref: ExternalPackageRef = ExternalPackageRef(category="PACKAGE-MANAGER")
+    second_ref = ExternalPackageRef(category="SECURITY")
+    package.add_pkg_ext_refs(first_ref)
+    package.add_pkg_ext_refs(second_ref)
+
+    # Not using write_anything here because we don't want to deal with validation
+    with open(temporary_file_path, "wb") as out:
+        writer = Writer(document, out)
+        writer.write()
+
+    parsed_document = parse_file(temporary_file_path)[0]
+    parsed_package: Package = parsed_document.packages[0]
+
+    assert len(parsed_package.pkg_ext_refs) is 2
+    parsed_reference_categories = list(map(lambda x: x.category, parsed_package.pkg_ext_refs))
+    assert first_ref.category in parsed_reference_categories
+    assert second_ref.category in parsed_reference_categories
+
+
 def minimal_document() -> Document:
-    document = Document()
+    document = Document(data_license=License.from_identifier('CC0-1.0'))
```

Comment:
This minimal document is nowhere near complete, adding the data license seems quite arbitrary

Reply:
I added exactly the amount of information that made the errors go away :sweat_smile: .
Tbh, I am not concerned about creating a "real" minimal working document at this stage. We should do that in the future (with the upcoming refactor) and use it in various test cases. But for now, I just wanted a quick working solution for these tests.

Reply:
ok then :)